src/libs/API.js

```diff
@@ -474,11 +474,17 @@ function GetIOUReport(parameters) {
 
 /**
  * @returns {Promise}
+ * @param {Array<String>} policyIDList
  */
-function GetPolicyList() {
+function GetPolicyList(policyIDList = []) {
+    if (!policyIDList || policyIDList.length > 1) {
+        LogUtil.warn('[API] Attempting to load multiple full policies at once', {policyIDList});
```

Comment:
This warning doesn't really make sense to me. What is its purpose? Is it to prevent calling this on multiple policies? If so, it doesn't accomplish that at all. If that's not the purpose, then I don't really know why we want this.

Reply:
> This warning doesn't really make sense to me. What is its purpose?

The intention is indeed that the warning should be logged whenever we attempt to load multiple policies at once.

> If so, it doesn't accomplish that at all.

Sorry, I'm not seeing how it fails to accomplish that. If the `policyIDList` is falsy, then the API will attempt to load all policies for the user. If the `policyIDList.length > 1`, then the user is attempting to load multiple full policies at once. Can you help me see what I'm missing here?

Reply:
Maybe a better approach is to log a warn in prod/stg but throw on dev?

Reply:
> The intention is indeed that the warning should be logged whenever we attempt to load multiple policies at once.

Haha, yes, I get that. What I mean is... who is going to look at the warning and what are they going to do about it? Just having a warning for the sake of having a warning doesn't do anybody any good.

Reply:
Umm, I am not entirely sure, but I think either warn or alrt that are logged in JS trigger a bugbot issue... (but we should check that because there's something that we ignore, it might be warns or both)

Reply:
> Maybe a better approach is to log a warn in prod/stg but throw on dev?

This makes sense, I wonder if this should be the approach for all `warn` and `alert` logs?

Also, my (unconfirmed) understanding was that an alert will trigger a bugbot issue and some >1 threshold of occurrences of a warning will trigger a bugbot issue.

Reply:
Looking [here](https://github.com/Expensify/Web-Expensify/blob/dc87516bdfa259cfd0084982d377380da8953e2e/script/bugBot.php#L356-L368) it seems the threshold is 10 occurrences in an hour. Not sure if that's warnings, alerts, or both

Reply:
And looking [here](https://github.com/Expensify/Web-Expensify/blob/b188c76fa748f334ca96f11529fa5cef89c09c48/lib/Log.php#L381-L437) it seems the behavior is the same for warnings and alerts. So >10 occurrences in an hour should create a bugbot issue.

Reply:
I updated `Log.warn` and `Log.alert` to only log server errors on staging or production. Check it out and let me know your thoughts!

Reply:
> This makes sense, I wonder if this should be the approach for all warn and alert logs?

Not sure, seems risky, but maybe would be good...  (for sure don't do that in this PR)

I was talking about this https://github.com/Expensify/Web-Expensify/blob/8d25fb6b9bcb86ab7d5a392cfaeb1ecc292bd1ca/lib/LogAPI.php#L50-L69

Reply:
> I updated Log.warn and Log.alert to only log server errors on staging or production.

Why?

Reply:
So we aren't creating bugbot issues from errors in people's local testing. I am fine removing that from this PR if you think it's risky.

Reply:
Would like to get this PR over the finish line so I just removed the `LogUtil.warn` and all the log changes for now.

Reply:
> So we aren't creating bugbot issues from errors in people's local testing. I am fine removing that from this PR if you think it's risky.

We wouldn't, we would record those errors in your local bugbot DB.